huxley/www/js/components/Huxley.js

```diff
@@ -51,21 +52,30 @@ var Huxley = React.createClass({
     var user = CurrentUserStore.getCurrentUser();
     if (User.isAnonymous(user)) {
       return (
+        <div>
```

Comment:
Is there a reason why you added this? Don't think it's needed here or for the others below